paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -151,37 +151,42 @@ internal fun Wallet(
     val padding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
 
     if (containerState.shouldShow) {
-        Column(modifier = modifier.padding(horizontal = padding)) {
-            if (containerState.showGooglePay) {
-                GooglePayButton(
-                    state = googlePayButtonState?.convert(),
-                    isEnabled = buttonsEnabled,
-                    onPressed = viewModel::checkoutWithGooglePay,
-                    modifier = Modifier.padding(top = 7.dp),
-                )
+        Column {
+            Column(modifier = modifier.padding(horizontal = padding - 4.dp)) {
```

Comment:
Why did these changes (and the nesting of `Column` inside `Column`) become necessary?

Reply:
The Google Pay button now comes with a 8.dp padding by default and you cannot remove it.

Reply:
That’s unfortunate. Can you add this note as a comment?

Still the question about the `Column` nesting though. Can we flatten this out?

Reply:
I can look into flattening it, yes